### ISSUE TITLE
Password Visibility Toggle Eye Icon Behaving Opposite

### ISSUE THREAD

**Issue by shivamgaur99** [Muskansahuincredible/StudyNotion-An-Online-Education-Platform#48]
The eye icon used to toggle the visibility of the password field is behaving in the opposite manner than expected. When the icon is clicked, the password visibility state is not changing correctly.

![image](https://github.com/Muskansahuincredible/StudyNotion-An-Online-Education-Platform/assets/128178418/70580b6d-21fd-4cc8-8bee-4559e87a6efe)

please assign it to me

**Comment by Abhinav-Sharma06** [comment]
Hi @shivamgaur99  assign me this task under GSSOC 2024 . I am confident that I can fix this bug.

**Comment by Muskansahuincredible** [comment]
@shivamgaur99 i dont think so there is a problem.. it is working fine ..will u please elaborate the issue

**Comment by shivamgaur99** [comment]
@Muskansahuincredible when you enter any password eye toggle should off like in this image
![image](https://github.com/Muskansahuincredible/StudyNotion-An-Online-Education-Platform/assets/128178418/3813ce45-f707-49af-8a2e-700fcd7f44b6)

**Comment by Aditi-Gopinath** [comment]
Please assign me this issue under GSSOC'24 @Muskansahuincredible

**Comment by PratikMane0112** [comment]
I am interested in the issue, can you please assign me.

**Comment by UtkarshBirla28** [comment]
@Muskansahuincredible  assign me this issue i will fix it tommorow only.
